### ISSUE TITLE
Problema na busca por contatos quando houver acentuação.

### ISSUE THREAD

**Issue by fernandodenardin** [ticketz-oss/ticketz#219]
Atualmente a busca por contatos leva em conta a acentuação. Ex. Se eu cadastrar "João" e na busca utilizar "Joao" o contato não é encontrado. Creio que tenha uma forma de fazer com que o banco de dados localize o contato tanto com acento quanto sem. Obrigado.

**Comment by leonardooliveira64** [comment]
seria interessante também colocar um campo, para escolher o que deseja pesquisar... no campo busca, se pesquisar pelo numero do contato, mensagem no ticket, protocolo... acredito que fica mais organizado...

**Comment by allgood** [comment]
@leonardooliveira64 , sim, porém esse ticket é da tela de contatos... onde só tem mesmo número e nome.

Pretendo reformular a tela de conversas, inclusive a parte de pesquisa

**Comment by allgood** [comment]
@fernandodenardin ... acredito que isso deve ser resolvido com a collation do banco de dados. vou pesquisar.
